modules/persistenceService/src/main/resources/application.conf

```diff
@@ -1,8 +1,8 @@
 dummy.jdbc {
     driver = "org.postgresql.Driver"
-    url = ""
+    url = "jdbc:postgresql:dummyproject"
     host = ""
     port = ""
-    user = ""
+    user = "sabasimi"
     password =""
```

Comment:
There's a cool feature of the property files that allow loading environment variables directly and I think this is a good case since we don't want to expose the specific details of our machine or test environment. This could be rewritten as:

```
    url = ""
    url = "${?DB_URL}"
    host = ""
    host = "${?DB_HOST}"
    port = ""
    port = "${?DB_PORT}"
    user = ""
    user = "${?DB_USERNAME}"
    password =""
    password ="${?DB_PASSWORD}"
```

Then, starting a SBT console with the following command will produce the same result:

```bash
env DB_URL=jdbc:postgresql:dummyproject DB_USERNAME= sabasimi sbt
```

See https://github.com/lightbend/config#optional-system-or-env-variable-overrides

Reply:
Interesting so this is like a `.env`. I will make the changes.